### ISSUE TITLE
Re implement utcTime()

### ISSUE THREAD

**Issue by twinstar26** [ethereum/aleth#5710]
In libdevcore/Common.cpp,
`time(0)` in `utcTime()` needs to be re implemented
https://github.com/ethereum/aleth/blob/22d02e9a7138f2a949e2ee48d207ce1a2ab7c53f/libdevcore/Common.cpp#L53-L59

**Comment by gill984** [comment]
Is the issue here that time(0) doesn't work on all platforms? Or maybe the specific way it is being called is unacceptable?

**Comment by twinstar26** [comment]
Yeah so, time(0) is a c style method. It's not that it doesn't work in all platforms but that now we have CPP chrono and boost clock, chrono, etc libraries. So I think we have to use chrono::system_clock maybe.?
Its being implemented here. Take a look for reference.(https://github.com/ethereum/aleth/blob/22d02e9a7138f2a949e2ee48d207ce1a2ab7c53f/libp2p/UDP.h#L66)
Thanks for your interest @gill984

**Comment by gill984** [comment]
Ah OK I see so replace the time.h method call with a call to chrono::system_clock like in the snippet you linked. Let's say I did that and wanted to unit test, is there a way to unit test without rebuilding the entire project?

No problem, I am new and trying to get into contributing to open source, so sorry if I ask obvious questions. Thanks for the help.

**Comment by twinstar26** [comment]
Yes you are right @gill984. No, you have to build the entire project. First build will take a lot of time but subsequent builds will be quicker due to caching mechanism. Also I think there is no need to make a unit test for this issue. Generally I have observed that we make unit test only after creating new function or extreme modification of existing function. Though you should test whether the changes you make are outputting appropriate return values using suitable tools of your choice. 
And yeah welcome to Open-Source. Please read  https://github.com/ethereum/aleth/blob/master/CODING_STYLE.md
https://github.com/ethereum/aleth/blob/master/CONTRIBUTING.md
And this for testing with existing unit tests
https://github.com/ethereum/aleth/blob/master/doc/usingtesteth.rst
For building read readme.md on the front page of GitHub project.

**Comment by twinstar26** [comment]
For any doubts please be in touch with @twinstar26 or more experienced people like @halfalicious, @gumb0 (maintainer) and others.
